docs/src/rules/array-bracket-newline.md

```diff
@@ -5,7 +5,7 @@ related_rules:
 - array-bracket-spacing
 ---
 
-This rule was **deprecated** in ESLint v8.53.0. Please use the corresponding rule in [`@stylistic/eslint-plugin-js`](https://eslint.style/packages/js).
+This rule was **deprecated** in ESLint v8.53.0. Please use the corresponding rule in [`@stylistic/eslint-plugin-js`](https://eslint.style/packages/js). Corresponding rule: [`array-bracket-newline`](https://eslint.style/rules/js/array-bracket-newline).
```

Comment:
Thanks for doing this. If we're going to including the corresponding rule, then I think it should be done like this:

```suggestion
This rule was **deprecated** in ESLint v8.53.0. Please use the [corresponding rule](https://eslint.style/rules/js/array-bracket-newline) in [`@stylistic/eslint-plugin-js`](https://eslint.style/packages/js).
```